.github/workflows/npm_publish.yml

```diff
@@ -0,0 +1,48 @@
+name: NPM publish on new release
+
+on: 
+  release:
+    types: [created]
+    branches: 
+      - master
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [15.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v2
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: npm ci
+    - run: npm run test
```

Comment:
Do we need to include the test here? If the tests fail will the publish fail? There shouldn't be code in master that fails our tests.

Reply:
ah I added it in as a safety check, but if we already verified that before pushing to master then I guess this step is not necessary.